docs/apis-tools/go-client/go-get-started.md

```diff
@@ -193,6 +193,8 @@ Add the following to `main.go` at the bottom of `func main()`.
 	fmt.Println(response.String())
 ```
 
+<!--- This is a bit of an anomaly. Thoughts? --->
```

Comment:
I'm not sure what this is referring to.

Reply:
We reference `order-process.bpmn` in the code snippet, but I think this is fine and shouldn't have any impact on the migration of this file.

Reply:
Won't be a problem.

There are bigger issues with consistency in this guide. I will create a separate issue.